WindowsServerDocs/storage/refs/refs-overview.md

```diff
@@ -98,6 +99,22 @@ Deploying ReFS as a backup target is best suited for applications and hardware t
 > [!NOTE]
 > Backup targets include the above supported configurations. Please contact application and storage array vendors for support details on Fiber Channel and iSCSI SANs. For SANs, if features such as thin provisioning, TRIM/UNMAP, or Offloaded Data Transfer (ODX) are required, NTFS must be used.
 
+### Performance considerations
+
+Pro:
+ReFS is an advanced filesystem over NTFS in many ways. It can greatly improve the performance in read / write operation scenarios. This is especially the case when comparing operations like creating Hyper-V virtual disks, snapshots or merging them. These performance benefits are not limited to Hyper-V though. 
```

Comment:
The way I read the first sentence, is that it is meant to compare ReFS to NTFS and find ReFS to be better. In such comparisons, it is necessary to be more precise about the difference, like e.g. (for example):
```suggestion
ReFS is a superior filesystem to NTFS in many ways. It can greatly improve the performance in read / write operation scenarios. This is especially the case when comparing operations like creating Hyper-V virtual disks, snapshots or merging them. These performance benefits are not limited to Hyper-V though. 
```
Again, my suggestion may be imprecise, mostly due to my lack of knowledge about this software technology. I am mostly trying to convey that in a comparison between two systems, where one is better than the other (in one or more ways), the comparison needs to include a qualifier to say how one of them is better than the other. When using the preposition "over" in a comparison, you often need to rephrase the preceding text to say how the preference is made or how one of them takes precedence over the other.

Reply:
Another way to phrase the first sentence could be:
```suggestion
ReFS is an advanced file system, superior to NTFS in many ways. It can greatly improve the performance in read / write operation scenarios. This is especially the case when comparing operations like creating Hyper-V virtual disks, snapshots or merging disks. These performance benefits are not limited to Hyper-V, though. 
```
I also threw in a suggestion for replacing "them", because "them" can be ambiguous here. Readability comma added near the end (natural speech). File System should most likely be separated and not in one word (in contrast to other germanic languages).